lib/dpl/provider/pypi.rb

```diff
@@ -64,7 +65,9 @@ def needs_key?
 
       def push_app
         context.shell "python setup.py register -r #{options[:server] || 'pypi'}"
```

Comment:
I should mention too, that Warehouse removes even the "you must have ran ``setup.py register`` at least once ever" caveat too.